### PR TITLE
Bug 1068874 - [Camera] Focus indicator graphic should be hidden from screen reader

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
Added aria-hidden="true" inside template in line 287

### CODE REVIEW

**Comment by try-server-hook** [comment]
[atiqueahmedziad Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9dace299f9ba)

File: apps/camera/js/views/viewfinder.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ module.exports = View.extend({
 
   template: function() {
     return '<div class="viewfinder-frame js-frame">' +
-        '<div class="viewfinder-video-container js-video-container">' +
+        '<div class="viewfinder-video-container js-video-container" aria-hidden="true">' +
```
Comment (wilsonpage) [inline apps/camera/js/views/viewfinder.js]: There's a linter fail: `apps/camera/js/views/viewfinder.js | line 287, col 90, Line is too long.`

Lines can't be longer that 80 characters. Please wrap it :)

Comment (wilsonpage) [inline apps/camera/js/views/viewfinder.js]: Sorry I should have been more clear:

```js
'<div class="viewfinder-video-container js-video-container" ' +
'aria-hidden="true">' +
```

**Comment by try-server-hook** [comment]
[atiqueahmedziad Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=92d2a69c81e8)

**Comment by atiqueahmedziad** [comment]
Thanks :) Done :)

**Comment by try-server-hook** [comment]
[atiqueahmedziad Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7928550e824e)

**Comment by try-server-hook** [comment]
[atiqueahmedziad Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2664ef85f27a)

**Comment by try-server-hook** [comment]
[atiqueahmedziad Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d39a9722967a)

**Comment by try-server-hook** [comment]
[atiqueahmedziad Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=af7480c0a27d)

**Comment by try-server-hook** [comment]
[atiqueahmedziad Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9fc109d3078b)

**Pull request closed without merging**